cleaners/teamviewer.xml

```diff
@@ -18,18 +18,40 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app TeamViewer
+    @url https://www.teamviewer.com/en/
+    @os Windows, Linux
+    @cleanerversion v1.5.0
+    @cleanerdate 2019-03-17
+    @cleanerby ??? & https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-13
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
 -->
 <cleaner id="teamviewer" os="windows">
   <label>TeamViewer</label>
+  <description>Remote control software</description>
+  <var name="profile">
+    <value search="glob" os="windows">$AppData\TeamViewer</value>
```

Comment:
FYI, I am committing to remove the unnecessary search="glob". Not a big deal, but it's not needed

Reply:
What do you mean with "committing to remove" ??? Sorry, maybe my English...
Should I remove it ??? I copied this from somewhere else...

Reply:
I made this commit https://github.com/bleachbit/bleachbit/commit/057694a08301b4939d59f7bed5cc996c94a1de52